content/support-matrix/delfin/add-driver-support/_index.md

```diff
@@ -0,0 +1,434 @@
+{{% /notice %}}
+
+9) To add all the support matrix tables content, copy lines from 57 to 432 from a file content/support-matrix/delfin/add-driver-support/_index.md 
+10) Fill appropriate driver data to support matrix tables. 
+11) Please find below sample driver support matrix data tables with sample data for reference to fill specific driver support matrix data.
+
+### Model and management software versions supported
+
+<table>
+   <thead>
+      <tr>
+         <th>Models</th>
+         <th>Management Software Versions</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Sample model name</td>
+         <td>Management software version</td>
+      </tr>
+      <tr>
+         <td></td>
+         <td></td>
+      </tr>
+   </tbody>
+</table>
+
+### Access information to register storage device
+
+<table>
+   <thead>
+      <tr>
+         <th>Access Type</th>
+         <th>Attributes</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td rowspan="4">REST</td>
+         <td>Host</td>
+         <td>Management software version host name</td>
+      </tr>
+      <tr>
+         <td>Port</td>
+         <td>Management software version port name</td>
+      </tr>
+      <tr>
+         <td>Username</td>
+         <td>Management software version Username</td>
+      </tr>
+      <tr>
+         <td>Password</td>
+         <td>Management software version Password</td>
+      </tr>
+      <tr>
+         <td rowspan="4">SSH</td>
+         <td>Host</td>
+         <td>Management software version host name</td>
+      </tr>
+      <tr>
+         <td>Port</td>
+         <td>Management software version port name</td>
+      </tr>
+      <tr>
+         <td>Username</td>
+         <td>Management software version Username</td>
+      </tr>
+      <tr>
+         <td>Password</td>
+         <td>Management software version Password</td>
+      </tr>
+      <tr>
+         <td></td>
+         <td></td>
+      </tr>
+   </tbody>
+</table>
+
+### Alert management information
+<table>
+   <tbody>
+      <tr>
+          <th>SNMP trap processing</th> 
+          <td>Specify whether SNMP trap is supported or not supported</td> 
+      </tr>
+      <tr>
+         <th>Query backend alert</th>
+         <td>Specify whether Query alert is supported or not supported</td>
+      </tr>
+      <tr>
+         <th>Clear backend alert</th>
+         <td>Specify whether Clear alert is supported or not supported</td>
+      </tr>
+      <tr>
+         <th>SNMP versions for trap</th>
+         <td>Specify SNMP supported version by driver</td>
+      </tr>
+      <tr>
+         <th>SNMP trap source</th>
+         <td>Alert source name supported by driver</td>
+      </tr>
+      <tr>
+         <th>Query alert source</th>
+         <td>Query alert source name supported by driver</td>
+      </tr>
+   </tbody>
+</table>
+
+### Performance registration information
+
+<table>
```

Comment:
This table is empty and is not formatted correctly. Please check

Reply:
updated sample data